internal/repository/task.go

```diff
@@ -0,0 +1,7 @@
+package internal
```

Comment:
Same here, the package should be the same as the folder...

Something like:
```suggestion
package repository
```